package.json

```diff
@@ -8,6 +8,7 @@
     "build:renderer": "cross-env NODE_ENV=production TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.renderer.prod.ts",
     "rebuild": "electron-rebuild --parallel --types prod,dev,optional --module-dir release/app",
     "lint": "cross-env NODE_ENV=development eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:fixall": "prettier --write '**/*.{js,jsx,ts,tsx,json,html,md,yml}' && prettier --write '**/.{eslintrc,prettierrc}'",
```

Comment:
I added this command so that we can easily fix everything in the future if this kind of issue occurs again.